File: POO - Python & Java/practica-2/gestorAplicacion/Jerarquia.py
from enum import Enum

class Jerarquia(Enum):
    INTERNO=("Interno"),
    PLANTA=("Medico de planta"),
    DIRECTORMEDICO=("Director medico")
    JEFE=("Jefe")

    _titulo=""

    def __init__(self,titulo):
        super().__init__()
        self._titulo=titulo

File: POO - Python & Java/practica-1/src/gestorAplicacion/Jerarquia.java
package gestorAplicacion;

public enum Jerarquia {
	INTERNO("Interno"),
	PLANTA("Medico de planta"), 
	DIRECTORMEDICO("Director medico"),    //ESTO LO PUSO BRAHIAN
	JEFE("Jefe");
	
	private String titulo;
	
	Jerarquia(String titulo) {
		this.titulo = titulo;
	}
	
	public String getTitulo() { return this.titulo; }
}
